src/transformers/__init__.py

```diff
@@ -380,7 +382,11 @@
         ReformerForQuestionAnswering,
         REFORMER_PRETRAINED_MODEL_ARCHIVE_LIST,
     )
-
+    from .modeling_mpnet import (
+        MPNetForMaskedLM,
+        MPNetModel,
+        MPNetForQuestionAnswering,
+    )
```

Comment:
Also other heads (`ForSequenceClassification`) should be supported. (You can do it later.)

Reply:
ForTokenClassification would be awesome for the sequence labeling people 😅

Reply:
Yeah, more = better